app/Check/Consumers/CertificateCheck.php

```diff
@@ -56,7 +59,15 @@ protected function doHardJob(\Pd\Monitoring\Check\Check $check): bool
 
 			try {
 				$curl = new \GuzzleHttp\Client();
-				$gradeResponse = \Nette\Utils\Json::decode((string) $curl->get($check->getSslLabsApiLink())->getBody(), \Nette\Utils\Json::FORCE_ARRAY);
+				$options = [
```

Comment:
Já bych na toho klienta a nastavení udělal extra factory, opakuje se ve všech kontrolách stejný kód, maximálně ten timeout udělat jako parametr.

Reply:
Souhlasím. Tento kód je rozkopírovanej ve všech kontrolách. 
Moje motivace je upravit to v samostaném PR, kde vyřeším továrnu na klienta a options jako objekt namísto pole. 
Tzn v prvním kroku jsem potřeboval doplnit ten timeout protože nám to dělá problémy.
Ve druhém kroku pak navrhnu nějaké PR, který můžeme ladit.